packages/database/src/migrations/20210610111044-AddReportForMosquitoSpecieesMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+const REPORT = {
+  id: generateId(),
+  code: 'PG_Strive_Mosquito_Species_MapOverlay',
+  config: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'facility',
+          },
+        },
+      ],
+      dataElements: ['STRVEC_AE-AT13'],
+    },
+    transform: [
+      {
+        "'name'": '$row.value',
+        transform: 'select',
+        "'numerator'": '1',
+        "'organisationUnitCode'": '$row.organisationUnit',
+      },
+      {
+        '...': ['numerator', 'organisationUnitCode', 'name'],
+        transform: 'select',
+        "'denominator'": 'sum($all.numerator)',
+      },
+      {
+        name: 'group',
+        numerator: 'sum',
+        transform: 'aggregate',
+        denominator: 'last',
+        organisationUnitCode: 'group',
+      },
+      {
+        '...': ['organisationUnitCode'],
+        "'value'": "'EXISTED'", // Just a tag to indicate we have data
+        '$row.name': 'fractionAndPercentage($row.numerator, $row.denominator)',
```

Comment:
I use `fractionAndPercentage` instead of `formatDataValueByFractionAndPercentage` because it is shorter.